pkg/kubelet/cm/event_dispatcher.go

```diff
@@ -91,6 +91,8 @@ func (ed *eventDispatcher) dispatchEvent(pod *api.Pod, cgroupPath string, kind l
 		Pod: jsonPod,
 	}
 
+	// TODO: support more than one isolator return list of replies
+	replies := []*lifecycle.EventReply{}
```

Comment:
Here's an alternative to think about: we could aggregate the replies by iteratively merging replies. That way we could return a single reply. That would simplify the logic in applyLimits. Opinions?